dev/travis/save-logs.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import sys
+import subprocess
+from datetime import datetime, timedelta
+
+def main(file, cmd):
+    print cmd, "writing to", file
+    out = open(file, "w")
```

Comment:
Thank you @Leemoonsoo ,but That will be fine that  "install.txt" be overwrited by next step output, 
because when  errors occur ```dev/travis/save-logs.py``` will print all the error info to console  and we could find them in travis-ci log .